content/user/ui/report/Org-stats.md

```diff
@@ -0,0 +1,51 @@
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed. 
+
+For more details on each category, select the **View** link under each heading. 
+
+## Licenses
+
+The Licenses tab features a Consumption trend chart of data for the past 12 months. Hover over a data point on the Assessment or Protection trend lines to see how may licenses were available that month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline of shows how many licenses are about to expire on given dates. For a different view of the data, take a look at the circular charts with breakdowns by fraction and percentage for Assessment and Protection. 
+
+If you have no Protection licenses, Contrast alerts you to the count of unlicensed servers.
+
+<a href="assets/images/org-stats-licenses.png" rel="lightbox" title="Organization statistics for licenses"><img class="thumbnail" src="assets/images/org-stats-licenses.png"/></a>
+
+<!-- Couldn't fit Assessment AND Protection view in browser window-->
+
+## Applications
+
+In the Applications tab, the Status thermometer chart shows the total number of applications broken down by the number that are licensed, unlicensed and archived. Click on the total number of onboarded applications to go to the Applications page for more details on each one. Contrast also remdinds you of how many licenses are available in your organization. 
+
+The circular Language Breakdown chart shows the number of applications by language in the inner band and by Technology or Grade in the outer band. Click the number of active applications to go to the grid in the Applications page. High Risk, Monitored and Expirations snapshots show the number of applications with critical open vulnerabilities, no protection and expiring licenses, respectively. Applications that you added within the last week and applications in an offline server are listed separately in the sidebar. 
+
+<a href="assets/images/org-stats-applications.png" rel="lightbox" title="Organization statistics for applications"><img class="thumbnail" src="assets/images/org-stats-applications.png"/></a>
+
+## Servers  
+
+Switch to the Servers tab to view a thermometer chart that shows the breakdown of all deployed servers by environment. Click on the total number of servers to go to Servers page for more information on each one. 
+
+The circular Containers Breakdown chart displays the number of deployed servers for each language in a given environment. Select a different environment in the dropdown menu to update the ring and total number of servers. Click on the total number of servers to go to the grid in the Servers page. Snapshots show servers being Assessed and Protected as well as all servers online compared to the total number of servers in the given environment. The right sidebar includes a list of newly onboarded, offline and deleted servers. 
```

Comment:
"Clicking on the server count will navigate you to the Servers page with the relevant environment filter applied."